### PR TITLE
[DDW-1082] Bump cardano-wallet to master and cardano-node to 1.34.1

### PR DESCRIPTION
**Pull request by jbgi**
* [Slack thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1650366876403389)

### CODE REVIEW

**Comment by michalrus** [comment]
The blocks start syncing on all 3 platforms, so I would assume this is correct.

Thank you, @jbgi! :bow: 

/cc @danielmain 

-----

Screenshot-20220408-124401
Screenshot-20220408-125030
Screenshot-20220408-130135

**Review by michalrus — APPROVED** [review]

**Review by danielmain — APPROVED** [review]

File: default.nix [hunk 1]
```diff
@@ -31,8 +31,14 @@ let
...
   };
   pkgs = import sources.nixpkgs { inherit system config; };
-  pkgsNative = localLib.iohkNix.getPkgsDefault {};
   sources = localLib.sources;
+  flake-compat = import sources.flake-compat;
+  walletFlake = flake-compat  { src = sources.cardano-wallet; };
+  walletPackages = with walletFlake.defaultNix.hydraJobs; {
+    x86_64-windows = linux.windows;
+    x86_64-linux = linux.native;
+    x86_64-darwin = macos.intel;
```
[the remark this answers is not in the document]
Comment (michalrus) [reply default.nix:40]: There is only `macos.silicon` in cardano-wallet’s `hydraJobs` – which is being added one line below:

```
% pwd
/home/mw/Development/cardano-wallet

% git rev-parse @
d3a5ad4b8e283b46ae5eb699eb68f4f4ce88de55

% nix repl
Welcome to Nix 2.4. Type :? for help.

nix-repl> :lf .
Added 24 variables.

nix-repl> outputs.hydraJobs.macos.<TAB><TAB>
outputs.hydraJobs.macos.intel    outputs.hydraJobs.macos.silicon
```

**Review by michalrus — CHANGES_REQUESTED** [review]
Wallet syncing doesn’t work:
* https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1650457031248759?thread_ts=1650363534.113549&cid=GGKFXSKC6
* https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1650455427271789?thread_ts=1650366876.403389&cid=GGKFXSKC6

Erroring out with `Cannot read property 'quantity' of undefined` in API responses.

**Review by michalrus — APPROVED** [review]
Thank you to all involved!

**Review by dmitrii-gaico — APPROVED** [review]

**Review by danielmain — APPROVED** [review]

**Pull request merged**